shared/templates/sebool/ansible.template

```diff
@@ -4,34 +4,44 @@
 # complexity = low
 # disruption = low
 
+{{% if product in ['ol8', 'ol9', 'rhel8', 'rhel9'] %}}
+{{% set PACKAGE_NAME = "python3-libsemanage" %}}
+{{% elif product == "sle15" %}}
+{{% set PACKAGE_NAME = "policycoreutils" %}}
+{{% else %}}
+{{% set PACKAGE_NAME = "libsemanage-python" %}}
+{{% endif %}}
+
+- name: Ensure {{{PACKAGE_NAME}}} installed
+  package:
+    name: "{{{ PACKAGE_NAME}}}"
+    state: present
+
+{{% if product == "sle15" %}}
+- name: Ensure list of additional packages are installed on SLE 15
+  become: yes
+  package:
+    name:
+      - policycoreutils-python-utils
+      - selinux-tools
+      - python3-selinux
+      - python3-semanage
+    state: present
+{{% endif %}}
+
 {{% if SEBOOL_BOOL %}}
 - name: Set SELinux boolean {{{ SEBOOLID }}} to {{{ SEBOOL_BOOL }}}
   seboolean:
     name: "{{{ SEBOOLID }}}"
     state: "{{{ SEBOOL_BOOL }}}"
     persistent: yes
+  when: ansible_facts.selinux.status == 'enabled'  
 {{% else %}}
 - (xccdf-var var_{{{ SEBOOLID }}})
-
-{{% if product in ['ol8', 'ol9', 'rhel8', 'rhel9'] %}}
-- name: Ensure python3-libsemanage installed
-  package:
-    name: python3-libsemanage
-    state: present
-{{% elif product == "sle15" %}}
-- name: Ensure policycoreutils installed
-  package:
-    name: policycoreutils
-    state: present
-{{% else %}}
-- name: Ensure libsemanage-python installed
-  package:
-    name: libsemanage-python
-    state: present
-{{% endif %}}
 - name: Set SELinux boolean {{{ SEBOOLID }}} accordingly
   seboolean:
     name: {{{ SEBOOLID }}}
     state: "{{ var_{{{ SEBOOLID }}} }}"
     persistent: yes
+  when: ansible_facts.selinux.status == 'enabled'  
```

Comment:
This test is used in two tasks. Maybe it would be simpler to use it once in a `block` and include the tasks inside this block. It would make the Playbook simpler and likely more performative.